integration/spark-common/src/main/scala/org/apache/carbondata/events/AlterTableEvents.scala

```diff
@@ -205,13 +205,15 @@ case class AlterTableCompactionAbortEvent(sparkSession: SparkSession,
 /**
  * Compaction Event for handling merge index in alter DDL
  *
- * @param sparkSession
- * @param carbonTable
- * @param alterTableModel
+ * @param sparkSession spark session
+ * @param carbonTable carbon table
+ * @param alterTableModel alter request
+ * @param segments optional list of segments to perform index merge
  */
 case class AlterTableMergeIndexEvent(sparkSession: SparkSession,
     carbonTable: CarbonTable,
-    alterTableModel: AlterTableModel) extends Event with AlterTableCompactionEventInfo
+    alterTableModel: AlterTableModel,
```

Comment:
AlterTableModel is already having `customSegmentIds`, why added a new argument. we can just use that ?

Reply:
ok, fixed